cmake/compiler/gcc/target_base.cmake

```diff
@@ -18,3 +18,52 @@ macro(toolchain_cc_cstd_flag dest_var_name c_std)
   set_ifndef(${dest_var_name} "-std=${c_std}")
 
 endmacro()
+
+#
+# The two macros are dumping grounds for toolchain-specific options
+# that were previously in the top-level CMakeLists.txt. This is merely
+# a stopgap until the toolchain abstraction work underway (by others)
+# is complete. (There are two of these to preserve the original order.)
+#
+
+macro(toolchain_cc_others_1)
+  zephyr_cc_option(-fno-asynchronous-unwind-tables)
```

Comment:
The indentation makes it look better, however if you drop it (it's not critical cause these blocks are neither long nor complicated) then you give `git blame` a chance not to point at you but the actual authors instead.